tf_encrypted/convert/register.py

```diff
@@ -40,6 +40,8 @@ def register() -> Dict[str, Any]:
         'flatten': flatten,
         'Slice': slice,
         'Neg': negative,
+        'Split': split,
+        'Identity': identity,
```

Comment:
I'm not sure about how we should handle these Identity ops long-term, since they're possible to prune out using various graph transformation tools from TF.  Let's keep it, but note we may change it in the future.

Reply:
good point.